pkg/client/unversioned/remotecommand/remotecommand_test.go

```diff
@@ -92,15 +92,21 @@ func fakeExecServer(t *testing.T, i int, stdinData, stdoutData, stderrData, erro
 
 		if len(errorData) > 0 {
 			fmt.Fprint(errorStream, errorData)
+		}
+		if errorStream != nil {
 			errorStream.Close()
 		}
 
 		if len(stdoutData) > 0 {
 			fmt.Fprint(stdoutStream, stdoutData)
+		}
+		if stdoutStream != nil {
 			stdoutStream.Close()
 		}
 		if len(stderrData) > 0 {
 			fmt.Fprint(stderrStream, stderrData)
+		}
+		if stderrStream != nil {
```

Comment:
If I remove the Close() calls, the tests just hang. The commit message explains why - if we don't close the error stream, another goroutine blocks forever waiting for more errors.

Does your PR fix that?